tests/Aggregate/ConfigurableAggregateTranslatorTest.php

```diff
@@ -363,26 +391,19 @@ public function it_uses_callback_to_convert_message_into_custom_domain_event()
 
         $historyEvents = new \ArrayIterator([$historyEvent->reveal()]);
 
-        $translator = new ConfigurableAggregateTranslator(null, null, null, null, null, null, function (Message $message) {
+        $this->configProphecy->messageToEventCallback()->willReturn(function (Message $message) {
             return ['custom' => 'domainEvent'];
         });
 
+        $translator = new ConfigurableAggregateTranslator($this->configProphecy->reveal());
+
         $ar = $translator->reconstituteAggregateFromHistory(AggregateType::fromAggregateRootClass(DefaultAggregateRoot::class), $historyEvents);
 
         $this->assertEquals([['custom' => 'domainEvent']], $ar->getHistoryEvents());
     }
 
     /**
      * @test
-     * @expectedException \InvalidArgumentException
-     */
-    public function it_forces_message_to_event_callback_to_be_a_callable()
-    {
-        new ConfigurableAggregateTranslator(null, null, null, null, null, 0);
-    }
-
-    /**
-     * @test
      * @expectedException Prooph\EventStore\Aggregate\Exception\AggregateTranslationFailedException
```

Comment:
remove `@expectedException` in phpdoc and use `$this->expectException(...)` instead.